core/src/main/java/org/apache/gravitino/EntityStore.java

```diff
@@ -55,16 +55,39 @@ public interface EntityStore extends Closeable {
    * <p>Note. Depends on the isolation levels provided by the underlying storage, the returned list
    * may not be consistent.
    *
-   * @param namespace the namespace of the entities
    * @param <E> class of the entity
+   * @param namespace the namespace of the entities
    * @param type the detailed type of the entity
    * @param entityType the general type of the entity
-   * @throws IOException if the list operation fails
    * @return the list of entities
+   * @throws IOException if the list operation fails
    */
   <E extends Entity & HasIdentifier> List<E> list(
       Namespace namespace, Class<E> type, EntityType entityType) throws IOException;
```

Comment:
What I mean is that these interface can be implemented like:

```java
default XX list(xxxx) {
    return list(xx, xx, xx, Lists.emptyList)
}
```

Reply:
Done.